### PR TITLE
Some improvements to the v5 page

### PR DESCRIPTION
**Pull request by NicEastvillage**
Fixes some typos and rephrases sentences.

Replaces the migration section with a reference to the Python language migration page as it is more up-to-date.

Also includes indentation fixes for all pages.

### CODE REVIEW

**Review by VirxEC — APPROVED** [review]

File: docs/framework/v5.md [hunk 1]
```diff
@@ -19,43 +19,44 @@ Any updates to RLBot v5 beta and language interfaces will be announced in the [R
...
+- Our reasoning for creating v5 and an overview of its architecture [->](#why-v5),
+- A list of features completed/reworked/todo/removed in v5 (beta) [->](#features-of-v5),
+- A guide on breaking changes and how to transition from v4 to v5 [->](#breaking-changes-migration),
+- Links to v5 language interfaces and more [->](#links-and-resources).
 
 
 ## Why v5?
 
-RLBot v4 has been the most userfriendly and stable version of RLBot yet.
+RLBot v4 has been incredible userfriendly, feature rich, and stable.
```
Comment (swz-git) [inline docs/framework/v5.md:30]: incredibly*

File: docs/framework/v5.md [hunk 2]
```diff
@@ -19,43 +19,44 @@ Any updates to RLBot v5 beta and language interfaces will be announced in the [R
...
 It was made in 2018 when RLBot was still using DLL injection.
-Later, in 2019, the collaboration with Psyonix and the official API were revealed.
+Later, in 2019, the collaboration with Psyonix and the official API were incorporated.
```
Comment (swz-git) [inline docs/framework/v5.md:32]: introduced may be better wording?

File: docs/framework/v5.md [hunk 3]
```diff
@@ -19,43 +19,44 @@ Any updates to RLBot v5 beta and language interfaces will be announced in the [R
...
 Since then, there have only been minor changes and new features, which proves how well-designed v4 was.
 In fact, v5 is not that different from v4.
 It is primarily a rewrite and an opportunity fix the flaws of v4 that have been revealed over the years.
 So here are the main reasons we made v5:
 
 - The closed-sourced part of v4 is doing more than just communication with the Psyonix API.
   This has long prevented the community from fixing certain bugs themselves.
-  Therefore we are extracting part of the closed-source part into the opens-source part of RLBot such that only the communication is closed-source.
```
Comment (swz-git) [inline docs/framework/v5.md:40]: opens-source -> open-source

File: docs/framework/v5.md [hunk 4]
```diff
@@ -19,43 +19,44 @@ Any updates to RLBot v5 beta and language interfaces will be announced in the [R
...
 - The closed-sourced part of v4 is written in C++.
   By rewritting the backend of RLBot in C# we hope that more people feel comfortable contributing to it.
 - No more DLLs. You may not be aware, but most bots still uses a DLL to communicate directly to the RLBot backend.
   In v5, the transition to web sockets is finally complete and this allows the bot inferfaces to be improved (e.g. Python type hints are not a hack anymore).
-- All languages can start matches in v4, technically, but the logic for launching of Rocket League and bots was only implemented in Python (and partially Rust).
-  This discourage us from writing match starters (GUI, autoleague, etc) in other languages than Python (even CleoPetra in Java uses a Python process to start matches).
-  Therefore we move this logic to the backend for v5 such that more languages can benefit from it.
+- All languages can start matches in v4, but the logic for launching of Rocket League and bots was only implemented in Python (and partially Rust).
+  This discourage people from writing match starters (GUI, autoleague, etc) in other languages than Python (even CleoPetra written in Java uses a Python process to start matches).
```
Comment (swz-git) [inline docs/framework/v5.md:46]: discourage -> discouraged

**Pull request merged**